packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerPlugin.java

```diff
@@ -29,12 +34,14 @@
 import java.util.List;
 
 /** Location Plugin */
```

Comment:
(optional since this was here before) Would you mind fixing (or removing) this incorrect doc string?

Reply:
Done.. maybe?